ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0113__dsc_components.robot

```diff
@@ -6,321 +6,548 @@ Resource            ../../../../../Resources/OCP.resource
+${ETCD_LABEL_SELECTOR}                                  component=model-mesh-etcd
+${ETCD_DEPLOYMENT_NAME}                                 etcd
+${ODH_MODEL_CONTROLLER_LABEL_SELECTOR}                  app=odh-model-controller
+${ODH_MODEL_CONTROLLER_DEPLOYMENT_NAME}                 odh-model-controller
+${MODELREGISTRY_CONTROLLER_LABEL_SELECTOR}              control-plane=model-registry-operator
+${MODELREGISTRY_CONTROLLER_DEPLOYMENT_NAME}             model-registry-operator-controller-manager
+${KSERVE_CONTROLLER_MANAGER_LABEL_SELECTOR}             control-plane=kserve-controller-manager
+${KSERVE_CONTROLLER_MANAGER_DEPLOYMENT_NAME}            kserve-controller-manager
+${TRUSTYAI_CONTROLLER_MANAGER_LABEL_SELECTOR}           app.kubernetes.io/part-of=trustyai
+${TRUSTYAI_CONTROLLER_MANAGER_DEPLOYMENT_NAME}          trustyai-service-operator-controller-manager
+${FEASTOPERATOR_CONTROLLER_MANAGER_LABEL_SELECTOR}      app.kubernetes.io/part-of=feastoperator
+${FEASTOPERATOR_CONTROLLER_MANAGER_DEPLOYMENT_NAME}     feast-operator-controller-manager
+${NOTEBOOK_CONTROLLER_DEPLOYMENT_LABEL_SELECTOR}        component.opendatahub.io/name=kf-notebook-controller
+${NOTEBOOK_CONTROLLER_MANAGER_LABEL_SELECTOR}           component.opendatahub.io/name=odh-notebook-controller
+${NOTEBOOK_DEPLOYMENT_NAME}                             notebook-controller-deployment
+${IS_NOT_PRESENT}                                       1
 &{SAVED_MANAGEMENT_STATES}
-...  RAY=${EMPTY}
-...  KUEUE=${EMPTY}
-...  CODEFLARE=${EMPTY}
-...  TRAINING=${EMPTY}
-...  DASHBOARD=${EMPTY}
-...  DATASCIENCEPIPELINES=${EMPTY}
-...  MODELMESHERVING=${EMPTY}
-...  MODELREGISTRY=${EMPTY}
-...  KSERVE=${EMPTY}
-...  TRUSTYAI=${EMPTY}
-...  WORKBENCHES=${EMPTY}
-...  FEASTOPERATOR=${EMPTY}
-
-@{CONTROLLERS_LIST}    kserve-controller-manager    odh-model-controller    modelmesh-controller
+...                                                     RAY=${EMPTY}
+...                                                     KUEUE=${EMPTY}
+...                                                     CODEFLARE=${EMPTY}
+...                                                     TRAINING=${EMPTY}
+...                                                     DASHBOARD=${EMPTY}
+...                                                     DATASCIENCEPIPELINES=${EMPTY}
+...                                                     MODELMESHERVING=${EMPTY}
+...                                                     MODELREGISTRY=${EMPTY}
+...                                                     KSERVE=${EMPTY}
+...                                                     TRUSTYAI=${EMPTY}
+...                                                     WORKBENCHES=${EMPTY}
+...                                                     FEASTOPERATOR=${EMPTY}
+
+@{CONTROLLERS_LIST}
+...    kserve-controller-manager
+...    odh-model-controller
+...    modelmesh-controller
 @{REDHATIO_PATH_CHECK_EXCLUSTION_LIST}    kserve-controller-manager
```

Comment:
My bad, you are now handling this using the separate `${KSERVE_IMAGE_PULL_PATH}` variable. So this can be removed now.

Reply:
ok I will remove